### ISSUE TITLE
Script cant bypass the bot protection

### ISSUE THREAD

**Issue by nevpo** [grrttedwards/wow-addon-updater#148]
**Describe the bug**
What are you seeing when you run the script?
Upon running `pipenv run python -m updater` all of my addons give this exception: ```Failed to find addon version number for game version: GameVersion.retail,```. 
After i checked the log it seems that Cloudflare changed the algorithm again, so i tried to update the cloudscraper library and looks like it doesnt exist anymore (the links to github repo and the author lead to 404).

**If you run `pipenv update` on the command line and try again, does the issue go away?**
No

**Log output**
Please include the contents of `addon-updater.log`, from the root of the directory.
Either copy-paste the contents, or attach the file to this issue.
[addon-updater.log](https://github.com/grrttedwards/wow-addon-updater/files/4863584/addon-updater.log)

**Comment by grrttedwards** [comment]
Hi @nevpo thank you for reporting! I made a change one of the recent releases that has been working well for quite some time, so I was a little sad to see your issue 😝 

I think what you saw was Curse having a transient problem, and the cloudscraper module getting confused and reporting it found a challenge it doesn't know how to handle. _I think_ that's what happened in your case. On my end it's working, and the automated tests are still passing since your issue was created. Try again and let me know what you find.

Also, I see the github source now 404's... that's not good... the package is still hosted on pypi, and even released a new version 16 hours ago, but now I'm wishing I had forked the source code when I had the chance...
